backend/controllers/groupController.js

```diff
@@ -87,12 +88,35 @@ const addGroupMember = async (req, res) => {
     res.status(200).json(group);
 }
 
+const removeGroupMember = async (req, res) =>{
+    const { id } = req.params;
```

Comment:
uncomment to activate the code

Reply:
it could be useful for the addGroupMember function if we wish to only have the creator of the group have those rights, not sure what you mean about "worth adding below for authentication". Which line of code were you mentioning?

Reply:
so it basically makes sure the person who is logged in is the only person who can enable this fucntion.